src/vs/workbench/parts/files/browser/fileActions.ts

```diff
@@ -2362,7 +2362,7 @@ export function keybindingForAction(id: string): Keybinding {
 			return new Keybinding(KeyMod.CtrlCmd | KeyCode.KEY_S);
 		case DeleteFileAction.ID:
 		case MoveFileToTrashAction.ID:
-			return new Keybinding(KeyCode.Delete);
+			return new Keybinding(isMacintosh ? KeyMod.CtrlCmd | KeyCode.Backspace : KeyMod.Shift | KeyCode.Delete);
```

Comment:
@SamVerschueren it should actually be just KeyCode.Delete on Windows and Linux. The shift+delete was added to have a way to bypass the trash when doing the delete.